scripts/export_datastore_variables.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-export DATASTORE_DATABASE_HOST=${DATASTORE_DATABASE_HOST:-postgresql}
+export DATASTORE_DATABASE_HOST=${DATASTORE_DATABASE_HOST:-postgres}
```

Comment:
This file is used entrypoint.sh for dev and not dev

Reply:
This should not be a problem, right? Because in dev setup, I explicitely given the database host, so this default value is not used.

Reply:
I'm not sure about this!